src/utilities/_hypre_utilities.hpp

```diff
@@ -120,8 +120,16 @@ struct hypre_device_allocator
 
 /* WM: problems with this being inside extern C++ {} */
 /* #include <CL/sycl.hpp> */
+<<<<<<< HEAD
+
+#endif // defined(HYPRE_USING_CUDA)
+
+#if defined(HYPRE_USING_ROCSPARSE)
+#include <rocsparse.h>
+=======
 #if defined(HYPRE_USING_ONEMKLSPARSE)
```

Comment:
looks like this hasn't been merged properly.

Reply:
good catch. Sorry about that. Fixed.